repl/src/main/scala/com/cloudera/livy/repl/PySparkAddPyFileJob.java

```diff
@@ -0,0 +1,55 @@
+package com.cloudera.livy.repl;
+
+import com.cloudera.livy.Job;
+import com.cloudera.livy.JobContext;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.FileSystem;
+import org.apache.hadoop.fs.LocatedFileStatus;
+import org.apache.hadoop.fs.Path;
+import org.apache.hadoop.fs.RemoteIterator;
+import org.apache.spark.SparkContext;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URI;
+
+public class PySparkAddPyFileJob implements Job<Object> {
+
+    private PySparkJobProcessor pySparkJobProcessor;
+    private String path;
+    private SparkContext sparkContext;
+
+    public PySparkAddPyFileJob(
+           String path,
+           PySparkJobProcessor pySparkJobProcessor,
+           SparkContext sparkContext) {
+        this.path = path;
+        this.pySparkJobProcessor = pySparkJobProcessor;
+        this.sparkContext = sparkContext;
+    }
+
+    @Override
+    public Object call(JobContext jc) throws Exception {
```

Comment:
This duplicates a bunch of code from `AddJarJob`. Maybe there should be a common base class for both to avoid that?